### DISCUSSION TITLE
Please add option to run py-editor code on page load.

### DISCUSSION THREAD

**Discussion by MarcSkovMadsen** [pyscript/pyscript#1909] [Proposals]
I would like to be able to embed one or more `py-editor`s on my web site. But I would not like my users to have to click the button as they would normally be reading the page and expect the output to load automatically. I just want my users to optionally be able to edit and reexecute the code.

## Additional Context

I want something that works a bit like Panels code + output blocks below. But with a live editor instead of static code.

![image](https://github.com/pyscript/pyscript/assets/42288570/0b486705-02bf-45fa-9903-f8e69fb3e592)

**Comment by WebReflection** [comment]
would an *autoplay* or *autorun* work as attribute?

**Reply by MarcSkovMadsen** [reply]
Yes.

**Reply by MarcSkovMadsen** [reply]
A thing that would be *nice to have* it the attribute documented including some documentation for how to create a relevant loading spinner while the page/ pyscript is loading.

**Reply by WebReflection** [reply]
we have various examples about that but it's true it's not in te docs yet ... planning to propose this ASAP today though
